### DISCUSSION TITLE
Do we have plan to support output plugin to submit Logs, Metrics, or Traces to an OpenTelemetry endpoint

### DISCUSSION THREAD

**Discussion by karthiknagraj** [fluent/fluentd#4167] [Ideas]
Currently, we have an output plugin for submitting Logs, Metrics, or Traces to an OpenTelemetry endpoint using fluentbit. Is there any plan to support a similar plugin for Fluentd?

**Comment by greaterthanmost** [comment]
Are there any updates on this?
